api/questions.test.js

```diff
@@ -20,6 +20,10 @@ describe('Questions API', () => {
         .post('/api/queues/1/questions')
         .send(question)
       expect(res.statusCode).toBe(201)
+      expect(res.body.name).toBe('a')
+      expect(res.body.location).toBe('b')
+      expect(res.body.topic).toBe('c')
+      expect(res.body.askedBy.netid).toBe('dev')
```

Comment:
Can you add `expect(res.body).toHaveProperty('askedBy')` to this and others? Should be a little more explicit than assuming that `askedBy` is a property 😛

Reply:
to just the ones i modified or to anything that expects a question back? and should i remove the line that checks that the value is correct?

Reply:
anything that expects a question! And definitely leave the correctness check in there. The test would fail as is if it were missing, this would just provide a slightly more meaningful error message if it failed.